src/realmd/AuthSocket.cpp

```diff
@@ -511,6 +513,13 @@ bool AuthSocket::_HandleLogonProof()
     if (!Read((char*)&lp, sizeof(sAuthLogonProof_C)))
         return false;
 
+    PINData pinData;
```

Comment:
I'd say add an empty line to the example above as well, this is not one of those files being several thousand lines long so we don't have to feel bad using a bit of space to keeping it better visually organized in my opinion.